src/pages/index.scss

```diff
@@ -135,33 +135,45 @@
 .hero {
   color: $color-grey-000;
 
+  &__logo {
+    text-align: center;
+    margin: 0 auto $mu250;
+    justify-content: center;
+    display: flex;
+    width: $mu400;
+    height: $mu400;
+    fill: $color-grey-000;
+  }
+
   &__title {
-    font-size: $mu500;
-    line-height: $mu500;
+    font-size: $mu400;
+    line-height: $mu400;
     letter-spacing: -0.086875rem;
```

Comment:
because you don't use `em` for your letter spacing, witch would be a relative value from the current font-size, you need to change that if you change the font-size. I would recommend you to use an `em` value.